### PR TITLE
Mnemonic

### PR DESCRIPTION
**Pull request by mbickers**
Added mnemonic functionality:
- Mnemnic package with testing
- CLI prints clipcode mnemonic

### CODE REVIEW

File: pkg/mnemonic/mnemonic_test.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+			word := vocabulary[partOfSpeech][i]
+			if len(word) == 0 || rune(word[0]) != letter {
+				t.Errorf("vocabulary[%q][%v] = %q; expected a word starting with %q", partOfSpeech, i, word, letter)
+				break
+			}
+		}
+	}
+}
+
+func isValidPhrase(phrase, acronym string) bool {
```
Comment (evanweissburg) [inline pkg/mnemonic/mnemonic_test.go:47]: Does this function correctly validate the number of words in a phrase?

File: pkg/mnemonic/mnemonic_test.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		}
+
+		if isValidPhrase(phrase, acronym) {
+			t.Logf("%q --> %q", acronym, phrase)
+		} else {
+			t.Errorf("%q --> %q is invalid phrase", acronym, phrase)
+		}
+	}
+
+	invalidAcronyms := []string{"", "ab1", "...", "/\\", "aaaaaaaa", "ffĀ", "šaa"}
```
Comment (evanweissburg) [inline pkg/mnemonic/mnemonic_test.go:79]: May want to add numbers as invalid test case. Also, may want to add upper case character strings or mixed case strings as valid test cases.

**Review by evanweissburg — CHANGES_REQUESTED** [review]

File: pkg/mnemonic/mnemonic_test.go (hunk 1 above)
Comment (mbickers) [reply pkg/mnemonic/mnemonic_test.go:47]: Good catch, I will add that test case.

File: pkg/mnemonic/mnemonic_test.go (hunk 2 above)
Comment (mbickers) [reply pkg/mnemonic/mnemonic_test.go:79]: Good catch, I will add those text cases.

**Pull request merged**